services/QuillLMS/client/app/bundles/Evidence/styles/headerStyling.scss

```diff
@@ -27,6 +27,9 @@
     border-right: 1px solid #0b9d88;
     .left-side {
       display: flex;
+      .quill-tooltip-trigger {
+        padding-top: 7px;
```

Comment:
I feel like this way of aligning items often ends up being a little brittle and getting wonky down the line -- is it possible to use flex and `align-items: center` or something here instead?